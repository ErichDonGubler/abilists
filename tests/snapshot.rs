#[test]
fn snapshot() {
    use abilists::AbiList;

    insta::assert_debug_snapshot!(AbiList::from_bytes(include_bytes!(
        "../deps/glibc-abi-tool/abilists"
    )));
}
