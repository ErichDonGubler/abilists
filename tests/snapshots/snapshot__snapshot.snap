---
source: tests/snapshot.rs
assertion_line: 5
expression: "AbiList::from_bytes(include_bytes!(\"../deps/glibc-abi-tool/abilists\"))"

---
Ok(
    AbiList,
)
